### ISSUE TITLE
Quickstart example does not work with fresh install

### ISSUE THREAD

**Issue by tiggerntatie** [bertcarnell/pysigfig#1]
After install with pip, (e.g. pip install pysigfig), attempting to execute the quickstart example results in:

```
traceback (most recent call last):
  File "test.py", line 3, in <module>
    import pysigfig as pysf
  File "/home/cabox/workspace/colabhomework/env/lib/python3.7/site-packages/pysigfig/__init__.py", line 1, in <module>
    from .functions import exp, exp2, exp10, ln, log, log1p, log2, log10
  File "/home/cabox/workspace/colabhomework/env/lib/python3.7/site-packages/pysigfig/functions.py", line 4, in <module>
    from src.pysigfig.number import Const, Float
ModuleNotFoundError: No module named 'src'
```

**Comment by bertcarnell** [comment]
Thanks.  I can reproduce this:

```
sudo docker run -t -i ubuntu /bin/bash
apt-get install python3 python3-pip
python3 -m pip install pysigfig
python3
```

```python
import math
import pysigfig
```

There is clearly something different with my development environment.

**Comment by bertcarnell** [comment]
Should be fixed in 7e04a00e311440371371be5aa713c6ceec29600a.  Added a workflow to periodically test this.

Thanks for the bug report.
